src/screens/shared/dashboard/StatCards.js

```diff
@@ -22,49 +35,175 @@ const styles = {
     textAlign: 'center'
   }
 };
-class StatCards extends React.PureComponent {
-  getCardStats = () => {
-    // TODO: Replace with Airtable Call
+class StatCards extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      farms: [],
+      ksFarms: [],
+      totalHarvests: [],
+      recentHarvests: [],
+      gapCertification: [],
+      numFarmReferred: '',
+      numKsGapAccepted: '',
+      percentKsGapCertified: '',
+      percentKsGapApplied: '',
+      percentGapCertified: '',
+      nsevpHarvestFarms: '',
+      totalHarvestsPounds: '',
+      percentGapApplied: ''
+    };
+  }
+
+  async componentDidMount() {
+    const farms = await getAllFarmsForFarmSearch();
+    const gapCertification = await getAllGAPCertificationsForStatCard();
+    const totalHarvests = await getAllTotalHarvestsForStatCard();
+    const recentHarvests = await getAllRecentHarvestLogsForStatCard();
+    const ksFarms = farms.filter(farm => farm.ksAffiliated);
+
+    const numGapCertified = gapCertification.filter(farm => farm.gapCertified); // in gap certification there is no ks affiliated column
+
+    const numFarmReferred = gapCertification.filter(
+      farm => farm.farmReferred === 'Complete'
+    ).length;
+
+    const numKsGapAccepted = gapCertification.filter(
+      farm => farm.gapAccepted === 'Complete'
+    ).length;
```

Comment:
Same issue as above, we need to access the number of KS-affiliated farms that are GAP accepted.